io/pci/dlpar.py

```diff
@@ -351,7 +351,7 @@ def changehwres(self, server, operation, lpar_id, lpar, drc_index, msg):
                    -o %s --id %s -l %s ' % (server, operation, lpar_id,
                                             drc_index)
         try:
-            cmd = self.run_command(cmd)
+            cmd = self.run_command(cmd, 3000)
```

Comment:
@harihare Can this timeout be added to `listhwres()` as well ?

Reply:
@narasimhan-v I dont think it is requried there. Bydefault time our for all run_command is 300  seconds. The same worked fine for the large system too.